lib/util/browserdetection.js

```diff
@@ -0,0 +1,45 @@
+/* globals chrome, navigator */
+'use strict';
+
+const { guessBrowser } = require('@twilio/webrtc/lib/util');
+
+function isAndroid() {
+  return /Android/.test(navigator.userAgent);
+}
+
+function isIOS() {
+  return /iPad|iPhone|iPod/.test(navigator.userAgent);
+}
+
+function isChromeIOS() {
+  return guessBrowser() === 'chrome' && isMobile() && isIOS();
+}
+
+/**
+ * Check whether the current browser is a mobile browser
+ * @param {string} userAgent
+ * @returns {boolean}
+ */
+function isMobile(userAgent) {
+  userAgent = userAgent || navigator.userAgent;
```

Comment:
I think we can remove passing userAgent and this line, since we are not following this pattern anyway on the other similar methods.